test/reserve/GoodReserveCDai.uniswap.test.ts

```diff
@@ -447,6 +454,63 @@ describe("GoodReserve - buy/sell with any token through uniswap", () => {
       goodReserve.sell(tokenA.address, sellAmount, 0, 0, NULL_ADDRESS)
     ).to.be.reverted;
   });
+  it("should price increase after buy when RR is not 100%",async() =>{
+    //Initialise new market maker due to other one's ownership transfered to goodreserve so we cant change its RR
+    const MM = await ethers.getContractFactory("GoodMarketMaker");
+
+    marketMaker = (await upgrades.deployProxy(MM, [
+      nameService.address,
+      999388834642296,
+      1e15
+    ])) as GoodMarketMaker;
+    await marketMaker.initializeToken(
+      cDAI.address,
+      "100", //1gd
+      "10000", //0.0001 cDai
+      "500000" //50% rr
+    );
+
+    await marketMaker.transferOwnership(goodReserve.address);
+    await setDAOAddress("MARKET_MAKER", marketMaker.address);
+
+    let reserveToken = await marketMaker.reserveTokens(cDAI.address);
+    let reserveRatio = reserveToken.reserveRatio;
+   
+    let beforeGdBalance = await goodDollar.balanceOf(founder.address);
+    let buyAmount = BN.from("500000000000000000000000") // 500k dai
+    await dai["mint(uint256)"](buyAmount);
+    let gdPriceBefore = await goodReserve["currentPrice()"]();
+    
+    dai.approve(goodReserve.address, buyAmount)
+    await goodReserve["buy(address,uint256,uint256,uint256,address)"](dai.address, buyAmount, 0, 0, NULL_ADDRESS)
+    let gdPriceAfter = await goodReserve["currentPrice()"]();
+    let laterGdBalance = await goodDollar.balanceOf(founder.address);
+    expect(beforeGdBalance.lt(laterGdBalance)) // GD balance of founder should increase
+
+    expect(gdPriceAfter.gt(gdPriceBefore)); // GD price should increase
+
+  })
+  it("should price decrease after sell when RR is not 100%",async() =>{
+    
+
+    let reserveTokenBefore = await marketMaker.reserveTokens(cDAI.address);
+    let reserveRatioBefore = reserveTokenBefore.reserveRatio;
+   
+
```

Comment:
- remove double empty lines to single empty line
- add line between tests (it(...))